### DISCUSSION TITLE
Mechanics Question

### DISCUSSION THREAD

**Discussion by AddAndy** [HgAlexx/Buffet#33] [General]
Hi I have a few questions about how this add-on behaves in specific situations:
1. Does the addon make bottled nethergon the best mana potion when in the correct zone?
2. For the longer fights (that last more than 2 minutes) where you can use multiple pots, what happens if you only have 1 best potion?
 ex. You have 1 super mana, and 1 major mana, you use the macro to trigger the super mana, but 2 min later (still in combat) you trigger the macro again, does it use a super mana pot? 
 3. For Demonic/Dark runes, what takes priority if you have both?
 4. Is there a way to select the 2nd best item?

**Comment by HgAlexx** [comment]
Hello,
1. It has priority over potion or same value or lower, but will not be selected if you have better
2. Nop, the macro cannot be updated in combat
3. The BoP one has priority
4. Not yet but I'm working on it ;) (which allow to solve 2. with custom macro)
